### DISCUSSION TITLE
add icloudpd

### DISCUSSION THREAD

**Discussion by antonov124** [tteck/Proxmox#2335] [Ideas]
Hello, is it posible to include the https://github.com/icloud-photos-downloader/icloud_photos_downloader ? 
greetings

**Comment by AlexGeek77** [comment]
> Hello, is it posible to include the https://github.com/icloud-photos-downloader/icloud_photos_downloader ? 
> 
> greetings

This is why I love this thread. I was looking into buying a spare Mac to run a script to pull down my photo library so I could archive it on my NAS and in Amazon Glacier. I didn't even think a program like icloud-photo-downloader existed. I'll spin up a container now to use this and help archive my photo collection which has been concerning me about its vulnerability recently.

**Reply by tteck** [reply]
Only thing needed as I see it is:

```
apt install pip
pip install icloudpd
```

**Reply by antonov124** [reply]
Okay  thx
